### ISSUE TITLE
Allow custom grouping of custom actions in action flyout

### ISSUE THREAD

**Issue by a-b-r-o-w-n** [microsoft/BotFramework-Composer#3455]
We need a way for users to group their custom actions in the flyout menu.

**Comment by yeze322** [comment]
Besides custom actions, those built-in action menu items shall be customizable as well.

**Comment by yeze322** [comment]
Here is the diagram. Action flyout will consume the `menu` part of uischema.
![image](https://user-images.githubusercontent.com/8528761/88154714-c8a9e100-cc39-11ea-8e19-47ca8c33498d.png)

**Comment by yeze322** [comment]
addressed in #3721

**Comment by yeze322** [comment]
#### User story - group a custom action
1. In ejected runtime, add a custom Action and run schema merge tool.
2. Now the custom $kind should appears in the flyout under the `Custom Action` label.
  ![image](https://user-images.githubusercontent.com/8528761/89153897-68e7fa00-d598-11ea-9dc9-2cf8f2ca4bed.png)
3. To group that $kind, update the `sdk.uischema` file. (under `BOT_NAME/schemas/sdk.uischema`)
  For example, to put it under `"Looping"`, the config is
  ```json
  {
    "menu": {
       "label": "My custom action",
       "submenu": ["Looping"]
    }
  }
  ```
  ![image](https://user-images.githubusercontent.com/8528761/89154079-ce3beb00-d598-11ea-8afc-1fdd2f32c0fe.png)
4. Now the custom $kind should appear under the assigned label ("Looping")
  ![image](https://user-images.githubusercontent.com/8528761/89154197-00e5e380-d599-11ea-820f-16ee46abd90b.png)
